docs/user/general-api.md

```diff
@@ -50,12 +50,13 @@ class SampleApplication : Application() {
         Glean.registerPings(Pings)
 
         // Initialize the Glean library.
-        Glean.initialize(applicationContext, uploadEnabled = true)
+        Glean.initialize(applicationContext, uploadEnabled = settings().isTelemetryEnabled)
```

Comment:
I'd be tempted to leave this line as it was or explain that `settings().isTelemetryEnabled` is something which is not part of the SDK but whatever contains the user preference/intent for having telemetry enabled.

Reply:
I think I prefer to add a comment about it -- the purpose of this change is to make it clear that this value should come from a user preference, and not be a constant (and also to match the Swift example below, but that's secondary)...

Reply:
Comment sounds good (maybe also in the swift example?)